build_wheels.py

```diff
@@ -13,12 +13,13 @@
 from typing import Union
 
 import requests
-import yaml
 from colorama import Fore
 from packaging.requirements import InvalidRequirement
 from packaging.requirements import Requirement
 
 from _helper_functions import print_color
+from yaml2requirement import merge_requirements
```

Comment:
Could you please move out merge_requirements from that module? It doesn't belong there. Also please rename the file based on the class defined there.